data/ru.json

```diff
@@ -719,5 +719,86 @@
 	"Зайгард",
 	"Дианси",
 	"Хуппа",
-	"Вулканион"
+	"Вулканион",
+	"Роулет",
+	"Дартрикс",
+	"Десиджуай",
+	"Литтен",
+	"Торракат",
+	" Инсинероар",
+	"Попплио",
+	"Бриан",
+	"Примарина",
+	"Пикипек",
+	null,
```

Comment:
I got the names from [Bulbapedia](http://bulbapedia.bulbagarden.net/wiki/Main_Page)
Didn't notice that some of the Russian names are missing 🤔